packages/alignments/src/SNPCoverageAdapter/SNPCoverageAdapter.ts

```diff
@@ -98,6 +99,7 @@ export default class extends BaseAdapter {
             basesPerSpan: bpPerPx,
           },
         )
+        // @ts-ignore
```

Comment:
why is ts-ignore needed?

Reply:
left in from when scoresToStats expected an Observable<BBIFeature> for feats, scoresToStats has since been changed but forgot to remove this and will be removed in next commit